### DISCUSSION TITLE
Is this provider available OpenTofu registry or Hashicorp registry only?

### DISCUSSION THREAD

**Discussion by jcamilo78** [SAP/terraform-provider-btp#615] [Q&A]
I team, 

I have a concern about is this stack will be available in opentofu that is opensource version of terraform due to community divorce, or just will be available in terraform(hashicorp)..?

Thank you in advanced.

**Comment by lechnerc77** [comment]
@jcamilo78 , currently the provider is available in the registry of Hashicorp only ((link)[https://registry.terraform.io/providers/SAP/btp/latest]) only. We are evaluating the technical compatbility with OpenTofu (see [issue](https://github.com/SAP/terraform-provider-btp/issues/460)).
Main technical blocker is beta state of the registry and the not yet defined process on how to add the provider to the registry [see corresponding issue [here](https://github.com/opentofu/registry/issues/104)]. Besides that there are several organizational topics that need to be sorted out. 

In short: as of today there are no plans or activities to add the Terraform provider for SAP BTP to the OpenTofu registry. This can change depending on our evaluation results. So there is no official support for OpenTofu as of today, However, this might change.
